HTML/1. intro.md

```diff
@@ -0,0 +1,31 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language, meaning that it is used to create webpages, so it's not like a normal programming language like Python
+
+___
+
+HTML is structured by tags, which make elements, here is an example of a tag - `<html>`
+
+And here is an example of an element `<html></html>`
+
+Each element has a opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `<` to begin and a `/>` to end. Everything else between the opening and closing tags is either content or other nested elements
```

Comment:
This is wrong. There are some elements which don't have an ending tag and there are some elements without an ending tag which have a '/>' in their starting tag / only tag itself.

Reply:
K forgot about that I'll add that, also It's me IntellectualGuy

Reply:
O hi. I didn't even realize, I thought you were some random dude lul. Thanks for adding a PR on this.

One note if you look at my C++ files you will see a pattern I use. Yours is pretty similar to that, but it would be nicer if you could use that same pattern. If you don't want to, its ok and I won't mind.

Reply:
Ok I'll check that out, also I don't mind if anyone else helps. And forgive if I ask anything stupid, I just joined github yesterday so I'm new to this.

Reply:
Do note that writing elements like "input" as `<input />` is incorrect, as that is XML syntax, not HTML, the correct way would be `<input>`, merely without an ending tag.